module/VuFind/src/VuFind/Regex/RegexFactory.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/**
+ * Factory for Regex
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Villanova University 2018.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Regex
+ * @author   Robby Roudon <roudonro@msu.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+
+namespace VuFind\Regex;
+
+use Laminas\ServiceManager\Exception\ServiceNotCreatedException;
+use Laminas\ServiceManager\Exception\ServiceNotFoundException;
+use Psr\Container\ContainerExceptionInterface as ContainerException;
+use Psr\Container\ContainerInterface;
+
+/**
+ * Factory for GetThisLoader
+ *
+ * @category VuFind
+ * @package  GetThis
+ * @author   Robby Roudon <roudonro@msu.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+class RegexFactory implements \Laminas\ServiceManager\Factory\FactoryInterface
+{
+    /**
+     * @var string Config file to use
+     */
```

Comment:
I think this would be more typical formatting for the project:

```suggestion
    /**
     * Config file to use
     *
     * @var string
     */
```